srcs/go/libkungfu-comm/cmds.go

```diff
@@ -2,14 +2,93 @@ package main
 
 import (
 	"os"
-
+    "bytes"
+    "encoding/json"
+    "net/http"
+    "strconv"
+    "net"
+    "context"
 	"github.com/lsds/KungFu/srcs/go/cmd/kungfu-run/app"
 )
 
 import "C"
-
+type Message struct {
+    Key string `json:"key"`
+}
+var httpc = http.Client{
+        Transport: &http.Transport{
+            DialContext: func(_ context.Context, _, _ string) (net.Conn, error) {
+                return net.Dial("unix","/tmp/http.sock")
+            },
+        },
+    }
 //export GoKungfuRunMain
 func GoKungfuRunMain() {
 	args := os.Args[1:] // remove wrapper program name (`which python`)
 	app.Main(args)
 }
+
+//export GoKungfuRunSendBegin
+func GoKungfuRunSendBegin() {
```

Comment:
you can abstract one API for the monitoring signal, and use different data to represent different signal types.

Reply:
Have changed